### ISSUE TITLE
4.0b1 - crash loop on start

### ISSUE THREAD

**Issue by kyods** [dirkwhoffmann/virtualc64#604]
4.0b1 keeps crash-looping upon pressing "Power" after adding the ROMs. The same set of ROMs worked fine with 3.4:

<img width="455" alt="Screenshot 2021-01-13 at 16 41 15" src="https://user-images.githubusercontent.com/66119492/104474388-3ff37880-55be-11eb-8892-f85c5a7a17e0.png">

The same situation happens when trying to power up after installing OpenRoms via the provided button - VirtualC64 goes into the crash loop until the folder VirtualC64 is deleted from Application Support.

**Comment by kyods** [comment]
Found the source of the problem: the crash loop occurs if multiple SIDs are selected in the configuration. With just one SID (default setup) VirtualC64 boots up just fine.

**Comment by dirkwhoffmann** [comment]
Could you run this example inside Xcode? I need to know where the emulator crashes.

**Comment by kyods** [comment]
Here you go:

```
RomConf.119::refreshRomTab()
HardwareConf.28::refreshHardwareTab()
HardwareConf.127::hwSidEnable(_:)
HardwareConf.28::refreshHardwareTab()
HardwareConf.127::hwSidEnable(_:)
HardwareConf.28::refreshHardwareTab()
HardwareConf.127::hwSidEnable(_:)
HardwareConf.28::refreshHardwareTab()
Assertion failed: (abs(l) < 0.15), function mixMultiSID, file /Users/kyods/Documents/virtualc64/Emulator/SID/SIDBridge.cpp, line 838.
Assertion failed: (abs(l) < 0.15), function mixMultiSID, file /Users/kyods/Documents/virtualc64/Emulator/SID/SIDBridge.cpp, line 838.
```

**Comment by dirkwhoffmann** [comment]
Thanks a lot, this explains it! It's a duplicate of #617.

Actually, the assertion  `abs(l) < 0.15` is not needed at all and I've already changed it to: 

 ```
        // Apply master volume
        l *= volL.current;
        r *= volR.current;
        
        // Apply ear protection
        assert(abs(l) < 1);
        assert(abs(r) < 1);
        
        stream.write(SamplePair { l, r } );
```

Background: macOS is supplied with sound samples of type float and the magnitude of these values must be very very low. To scale the values properly, I am multiplying with 0.0000125 at some point. While working on  multi SID support, it happened that I accidentally deleted this scaling factor and the effect was that my cat stood 1.5 meters in the air when I started a test run. Since then I though it's a good idea to add a guard of some kind.
